### ISSUE TITLE
Align the CSS or style configuration so that all documentation components (Docs, plugin docs, blueprints) use the same markdown style

### ISSUE THREAD

**Issue by anna-geller** [kestra-io/kestra#5629]
### Feature description

Currently things like info blocks differ for Docs and markdown in plugin docs:

![image](https://github.com/user-attachments/assets/3b1744ef-73c8-47e8-b791-74bb86dbc518)

plugin docs:

![image](https://github.com/user-attachments/assets/42c856cc-9d49-412a-8578-d86763c7f282)

We should use the same markdown style everywhere for consistency - ideally, let's follow the one used in Docs

**Comment by Nico-Kestra** [comment]
You can use [this](https://www.figma.com/design/ew0uXk0NRXJ2NBBJTNe2n1/01_UI?node-id=3595-34343&t=E8dQjzY99X5IrPNy-1) 

![Alert message](https://github.com/user-attachments/assets/a4175b42-0e26-4c38-8f29-9548da63fce1)

**Comment by talamo13** [comment]
Can I please be assigned this issue? I am a beginner and very interested in contributing.

**Comment by MilosPaunovic** [comment]
Go for it @talamo13! :rocket:

**Comment by SarthakBorude** [comment]
this seems to be an interesting issue ! if it hasnt been resolved till now I'm interested to work on it.

**Comment by SarthakBorude** [comment]
I have started working on this issue but I'm little doubtful if I'm doing it right. if possible please provide me little more context about this issue. it would be really helpful.

**Comment by MilosPaunovic** [comment]
What seems to be the issue you're having trouble with @SarthakBorude?

**Comment by SarthakBorude** [comment]
@MilosPaunovic actually I'm a little confused about which markdown style is mentioned in the issue.If possible please provide me more details. So I can work accordingly.

**Comment by anna-geller** [comment]
the style used for markdown

**Comment by Piyush-r-bhaskar** [comment]
Hi @anna-geller @MilosPaunovic ,

Sorry to jump in. I was just exploring this.

I believe plugin docs attached image text - _This feature has been moved to a new location_ is from BE and icon used is just an emoji. that's why this is not inline with Docs markdown.

![image](https://github.com/user-attachments/assets/c782080a-9848-4399-9407-ccd143a1098e)

For icon to appear in Docs as per @Nico-Kestra design May be we need to make some changes in `Alert.vue`.


**Just asking if there is need of icon for now or can this be only text ? Then, we can just remove the emoji for some of plugin docs alert.** 

![image](https://github.com/user-attachments/assets/014639ae-76d0-44f1-9813-0bc8c4d61756)

**Comment by Piyush-r-bhaskar** [comment]
> Hi @anna-geller @MilosPaunovic ,
> 
> Sorry to jump in. I was just exploring this.
> 
> I believe plugin docs attached image text - _This feature has been moved to a new location_ is from BE and icon used is just an emoji. that's why this is not inline with Docs markdown.
> 
> ![image](https://private-user-images.githubusercontent.com/102078527/395900523-c782080a-9848-4399-9407-ccd143a1098e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQyOTk3MzUsIm5iZiI6MTczNDI5OTQzNSwicGF0aCI6Ii8xMDIwNzg1MjcvMzk1OTAwNTIzLWM3ODIwODBhLTk4NDgtNDM5OS05NDA3LWNjZDE0M2ExMDk4ZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIxNVQyMTUwMzVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zZTdjNjE1ZmMyZTdiYTUzN2NiMTBhOGFiOTY4NDk3MGUzOGExMDIzYzg2YjA5OGRmODM0ZmI0YzE0MGIyMTAwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.aHqZ1d01ap3m7A781mLDsIF6uVdgc2whPg2vG1FKlLI)
> 
> For icon to appear in Docs as per @Nico-Kestra design May be we can make changes here for svg. ![image](https://private-user-images.githubusercontent.com/102078527/395900736-d803cd45-fae1-46e6-8983-85b5b7ad4a52.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQyOTk3MzUsIm5iZiI6MTczNDI5OTQzNSwicGF0aCI6Ii8xMDIwNzg1MjcvMzk1OTAwNzM2LWQ4MDNjZDQ1LWZhZTEtNDZlNi04OTgzLTg1YjViN2FkNGE1Mi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIxNVQyMTUwMzVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05MWM0ZDY5Mzk0NjlmODllYmRmYzgzNmFjNjgzY2NlMmI5ZmU0ZTk4MTZjYTY4ODYzZDRkMGUzY2FkZDI3N2RiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.ltoXJuURiPKrB17iZJFqKn5PUKm9DaMh_FwNhgUCdwo)
> 
> **Just asking if there is need of icon for now or can this be only text ?**
> 
> ![image](https://private-user-images.githubusercontent.com/102078527/395901116-014639ae-76d0-44f1-9813-0bc8c4d61756.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQyOTk3MzUsIm5iZiI6MTczNDI5OTQzNSwicGF0aCI6Ii8xMDIwNzg1MjcvMzk1OTAxMTE2LTAxNDYzOWFlLTc2ZDAtNDRmMS05ODEzLTBiYzhjNGQ2MTc1Ni5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIxNVQyMTUwMzVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kMTExZTlkZmU2ZDA4ZTEwZDhmM2ZkMGM4YjZjZWZhYmEwNDUzZDhhNjE4ZDExOTYzNzMwYzIxNTE1ODA2NTRlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.FzeKV-K1K55O4X_B2TE8mJ8jMLfE2aBKkn4Suc2oLG4)

How this will look of course need some styling ? 
![image](https://github.com/user-attachments/assets/ec9c552f-b135-4359-81a5-23ed4b43aa8a)

**Comment by anna-geller** [comment]
The icon in the blue info notice should also be blue ideally (not black). big thanks for picking up this topic @Piyush-r-bhaskar! 

![image](https://github.com/user-attachments/assets/a0ec10e4-8ce9-4796-9cf7-e1f15d519da1)

**Comment by Piyush-r-bhaskar** [comment]
Hi @anna-geller & @MilosPaunovic,

How you liked it ?

https://github.com/user-attachments/assets/f6104296-8c14-4484-8ac5-a611a212b1bf

https://github.com/user-attachments/assets/9870711a-75d0-43c5-93c4-4157abfa1c8d


As you know DOCs are using Bootstrap Alerts (not `El-Alert` like other places) defined in `@kestra-io\ui-libs\src\components\content\Alert.vue`. So I need to make the change for this in kestra UI libs _(currently I was doing `npm run dev` so see the change at my side_) So that DOCS markdown styling can be applied across platforms & also minor change in `kestra\ui\src\components\docs\DocsLayout.vue`.

![image](https://github.com/user-attachments/assets/d2f5505d-8590-4644-90e2-7b3c384fafa7)

**Your suggestions please ?**

**Also, any ideas from current assignees most welcome.** @SarthakBorude @talamo13

**Comment by MilosPaunovic** [comment]
Heads up @Piyush-r-bhaskar, this will have to wait for @anna-geller's input, but as today is her last day before holidays, we might postpone it for a bit, as it's lower priority for now.

**Comment by Piyush-r-bhaskar** [comment]
> Heads up @Piyush-r-bhaskar, this will have to wait for @anna-geller's input, but as today is her last day before holidays, we might postpone it for a bit, as it's lower priority for now.

Sure @MilosPaunovic 😊

**Comment by MilosPaunovic** [comment]
@anna-geller We might need you input here to proceed.

**Comment by anna-geller** [comment]
what inputs do you need from my side?

seeing the video, the only suggestion for improvement I could offer would be to place the icon at the top instead of in the middle - perhaps this might feel a bit friendlier?

![image](https://github.com/user-attachments/assets/b6f3ad7b-1e3b-4606-9f87-8db686f85452)

**Comment by MilosPaunovic** [comment]
@Piyush-r-bhaskar Does that answer you questions or do you need something else to proceed?

**Comment by Piyush-r-bhaskar** [comment]
Sure @anna-geller  That would be great. Thank you.

Yes @MilosPaunovic  Thank You.

So, I want to proceed like below:

- DOCs are using Bootstrap Alerts defined in `@kestra-io\ui-libs\src\components\content\Alert.vue`. So I need to make little change for this in `kestra UI libs` to add icons and alignment with content. 
- **Once this is merged**, I will be working under the hood for colors for alert-level.  **I want DOCS markdown styling can be applied across platforms means the product and kestra website** - So, I will try to handle the color styling with bootstrap class outside the library (_because website will only need dark theme colors for now but product will need both light and dark_).
-  I would first do the changes for website to be merged first and simultaneously for Product & merge after in few hrs after review.

Please I would love your suggestion for making it more perfect and clean. or Of course during review too 😃

**Most probably I will open a PR for this before Friday because currently have a Major Release on Tuesday (Hopefully it will go as per Go/No Go Assessment 😁)**

**Comment by MilosPaunovic** [comment]
Pinging @elevatebart, as he might be better suited to review this once the PR is opened, as reccently he's been doing work around both `ui-libs` and coloring scheme.
